packages/dx-react-grid-bootstrap4/src/templates/table-summary-item.test.jsx

```diff
@@ -17,6 +23,7 @@ describe('TableSummaryItem', () => {
   it('should pass style to the root element', () => {
```

Comment:
There isn't any code that merges styles for a summary item. Well, we can have a more common test here. Something like `should pass rest props to the root element`.